src/launch_headless.lua.t

```diff
@@ -0,0 +1,5 @@
+##lua-debug
+@launch_headless_instance_with_clean_environment+=
+local clean_args = { vim.v.progpath, '-u', 'NONE', '-i', 'NONE', '-n', '--embed', '--headless' }
+nvim_server = vim.fn.jobstart(clean_args, {rpc = true})
+vim.fn.rpcrequest(nvim_server, 'nvim_exec_lua', 'vim.o.runtimepath = (...)', { vim.o.runtimepath })
```

Comment:
I took the liberty to change this line. Would that work @dmitmel ?

Reply:
Yes, it does. Checked on my machine. Works with system Nvim and appimage Nvim. By the way, I think the parens around `...` are unnecessary (from what I understand about Lua's syntax).

Reply:
Nah, don't worry, do that yourself please